core/modules/comment/comment.module

```diff
@@ -2336,6 +2357,87 @@ function comment_file_download_access($field, $entity_type, $entity) {
   }
 }
 
+/**
+ * Implements hook_cron().
+ */
+function comment_cron() {
+  _comment_close_all();
+}
+
+/**
+ * Close all comments that meet the current threshold.
+ *
+ * @param $now
+ *  A UNIX timestamp for the expiration time.
```

Comment:
Could we elaborate on this? Its not clear what passing in a value would accomplish, or that there is a default.